cerulean.go

```diff
@@ -11,9 +11,16 @@ import (
 // Cerulean holds the handlers and port to instantiate the mock server
 type Cerulean struct {
 	// Addr is the address that Cerulean should listen at, eg: 127.0.0.1:51234
-	Addr          string
-	Handlers      map[string]http.Handler
-	Mux           *http.ServeMux
+	Addr           string
+	Handlers       map[string]http.Handler
+	Inventory      Inventory
+	Mux            *http.ServeMux
+	SubscriptionID string
```

Comment:
Are we gonna support multiple sub IDs at any point? Or is that a "table" discussion until we cross that bridge?

Reply:
There has to be a top level sub ID as far as I understand. I don't think that is a public endpoint as far as I know, this was the best way I could think of to emulate the signup process.

Reply:
Yup. NVM.